secant/Screens/Onboarding/OnboardingScreen.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  OnboardingScreen.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 9/17/21.
+//
+
+import SwiftUI
+
+protocol OnboardingScreenRouter: AnyObject {
+    func onboardingDone()
+}
+
+struct OnboardingScreen: View {
+    @ObservedObject var viewModel: OnboardingScreenViewModel
+
+    @State var router: OnboardingScreenRouter?
+
+    var body: some View {
+        VStack(
+            alignment: .center,
+            spacing: 30
+        ) {
+            Image(systemName: viewModel.currentStep.imageName)
+                .resizable()
+                .frame(
+                    width: 100,
+                    height: 100,
+                    alignment: .center
+                )
+
+            if let title = viewModel.currentStep.title {
+                Text(title)
+                    .font(.title)
+            }
+
```

Comment:
It would help when reviewing this code if we were consistent with how we introduced whitespace.  Above on line 36 we have a line of whitespace that lets me know that the image is done, but here below on 37 38 and 39 we don't have any to separate out those views.  Can we try adding white space around those views so we can easily see where one starts and one stops?

Reply:
I can add this guideline to the docs as well. Something along the lines:

Whitespace: SwiftUI body builders.
Different View definitions should be delimited by whitespace.